src/TensorOperations.jl

```diff
@@ -111,30 +111,31 @@ function TO.tensoralloc_contract(TC, A::ParametrisedTensorMap, pA::Index2Tuple,
                               B::AbstractTensorMap, pB::Index2Tuple, conjB::Bool,
                               pAB::Index2Tuple, istemp::Val=Val(false),
                               allocator=TO.DefaultAllocator())
-    coeffs = convert(Vector{Union{Number, Function}}, fill(t -> 0, length(A)))
-    tensors = fill(TO.tensoralloc_contract(TC, A.tensors[1], pA, conjA, B, pB, conjB, pAB, istemp, allocator), length(A))
-    tensors .+= fill(ones(TC, space(tensors[1])), length(tensors))
-    return ParametrisedTensorMap(tensors, coeffs)
+    tensors = map(eachindex(A)) do i
+        return TO.tensoralloc_contract(TC, A.tensors[i], pA, conjA, B, pB, conjB, pAB, istemp, allocator)
+    end
```

Comment:
It's a bit strange to map over the indices:
```julia
tensors = map(A.tensors) do a
    return TO.tensoralloc_contract(TC, a, pA, conjA, B, pB, conjB, pAB, istemp, allocator)
end
```

Reply:
https://github.com/VictorVanthilt/ParametrisedTensorKit.jl/commit/3066c59d18f254ad9ddd3bff949e0d5374a52e79